app/src/main/java/gov/va/vro/security/ApiAuthKeyManager.java

```diff
@@ -27,17 +31,48 @@ public void setApiAuthKeys(ApiAuthKeys apiAuthKeys) {
     this.apiAuthKeys = apiAuthKeys;
   }
 
+  public JwtValidator getJwtValidator() {
+    return jwtValidator;
+  }
+
+  @Autowired
+  public void setJwtValidator(JwtValidator jwtValidator) {
+    this.jwtValidator = jwtValidator;
+  }
+
   @Override
   public Authentication authenticate(Authentication authentication) throws AuthenticationException {
 
-    String principal = (String) authentication.getPrincipal();
+    String authorizationHdr = (String) authentication.getPrincipal();
+    log.info("Header Information : " + authorizationHdr);
```

Comment:
You may want to change it to log.debug, as showing the token in log may be complained by security scanning.

Reply:
Good catch!